src/block/tab-labels/edit.js

```diff
@@ -592,6 +617,34 @@ const Edit = props => {
 					</InspectorStyleControls>
 
 					<BlockDiv.InspectorControls />
+
+					<InspectorAdvancedControls>
+						<PanelAdvancedSettings
+							title={ __( 'Tab Anchors', i18n ) }
+							id="tabAnchors"
+						>
+							{ props.attributes.tabLabels.map( ( tab, index ) => (
```

Comment:
Add a help note before the fields saying "Add unique names to each tab, so you'll be able to link directly and open each tab."